### ISSUE TITLE
[Suggestion] Button on world generation screen

### ISSUE THREAD

**Issue by steve-the-player** [MaxenceDC/sparsestructures#11]
It would be nice to be able to change the worldgen options so that instead of Generate structures being an off/on toggle, it instead is off/sparse/regular/lots or something like that.

**Comment by MaxenceDC** [comment]
Yeah, you're right, I think that's a neat way to implement UI and very user-friendly. It's not the easiest thing to implement, especially since I don't really have experience in Minecraft modding... I'll look into it when I have time
Thank you!

**Comment by steve-the-player** [comment]
I'm not entirely certain, but I think some other mods that change the worldgen screen use the [Midnight Library](https://modrinth.com/mod/midnightlib)

**Comment by MaxenceDC** [comment]
Yeah, but I'd like to see if I'm able to keep my mod without any dependencies and try to implement it myself... (or maybe find a jarinjar lib or whatever it's called)

**Comment by steve-the-player** [comment]
I can respect that
